src/compat/array/sortedIndexBy.ts

```diff
@@ -25,7 +27,7 @@ const MAX_ARRAY_INDEX = MAX_ARRAY_LENGTH - 1;
 export function sortedIndexBy<T, R>(
   array: ArrayLike<T> | null | undefined,
   value: T,
-  iteratee?: Iteratee<T, R>,
+  iterateeFunc?: Iteratee<T, R>,
```

Comment:
```suggestion
  iteratee?: Iteratee<T, R>,
```
Could you update the parameter name like this? Also, isn't it fine to write `iterateeToolkit(iteratee)` below without using a ternary operator?